app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -301,4 +330,30 @@ private File createModelDownloadFile(File modelFile){
         }
         return modelDownloadFile;
     }
+
+    /**
+     * Create the Avatar file for a Student using a StudentImage
+     * @param studentImage
+     * @return
+     */
+    private File createAvatarFileFromStudentImage(StudentImage studentImage){
+        String dateFormatted = (String) DateFormat.format("yyyy-MM-dd_HHmmss", Calendar.getInstance());
+        String imageFilePath = StudentHelper.getStudentAvatarDirectory() + "/" + DeviceInfoHelper.getDeviceId(context) + "_" + dateFormatted + ".png";
+        File avatarFile = new File(imageFilePath);
+        try {
+            Log.i(getClass().getName(), "createAvatarFileFromStudentImage: Preparing InputStream and OutputStream to copy the StudentImage into the Avatar directory");
+            Log.i(getClass().getName(), "createAvatarFileFromStudentImage: InputStream with the file: " + studentImage.getImageFileUrl());
+            InputStream inputStream = new FileInputStream(studentImage.getImageFileUrl());
```

Comment:
@sladomic What is the resolution of the image? Can it be used for example in the chat application (96dp x 96dp)? https://github.com/literacyapp-org/literacyapp-chat/blob/master/app/src/main/res/layout/activity_chat_list_item.xml#L8

![screenshot_20161218-143629](https://cloud.githubusercontent.com/assets/1451036/21293247/97120bbe-c52f-11e6-83a5-e489a74bcb7d.png)